### PR TITLE
Fixes #50. Adding list of workspaces in collection views. It also sho…

### LINKED ISSUE #50: Collection Show/Edit View Changes
- [x] Show tile/list of workspaces in show and edit view
  * Look into removing edit/destroy buttons on show view

### PR DESCRIPTION
**Pull request by versae**
…w edit links when appropriate

### CODE REVIEW

File: app/controllers/collections_controller.rb [hunk 1]
```diff
@@ -21,6 +22,7 @@ def new
 
   # GET /collections/1/edit
   def edit
+    @is_edit = true
```
Comment (jkeck) [inline app/controllers/collections_controller.rb]: This `@is_edit` boolean seems off to me.

Perhaps this is the point that we should start diverting our show and edit partials and be more explicit about this.  I'm more than happy to talk through other solutions if it seems like we would be duplicating too much boilerplate templating code (although I'm not sure we know that now).

Comment (versae) [inline app/controllers/collections_controller.rb]: Sure. I'd be happy to know what's best.

File: spec/views/collections/index.html.erb_spec.rb [hunk 2]
```diff
@@ -8,5 +8,11 @@
   it 'renders a list of collections' do
     render
     expect(rendered).to have_css 'td', text: /My Collection/, count: 2
+    expect(rendered).to have_css 'a', text: /New Workspace/, count: 2
+  end
+
+  it 'renders a links to create workspaces from each collection' do
+    render
+    expect(rendered).to have_css 'a', text: /New Workspace/, count: 2
```
Comment (jkeck) [inline spec/views/collections/index.html.erb_spec.rb]: I think we may have some other commits in this branch?  Can you rebase these out?  These are going to conflict with changes in #67 (as I had to fix these up after rebasing the PR that introduces these lines).

File: spec/views/collections/edit.html.erb_spec.rb [hunk 3]
```diff
@@ -2,12 +2,17 @@
...
+    @collection = assign(:collection, create(:collection, workspaces: [create(:workspace)]))
   end
 
   it 'renders the edit collection form' do
     render
     expect(rendered).to have_css 'input[name="collection[name]"]'
     expect(rendered).to have_css 'input[type="submit"]'
   end
+
+  it 'renders the related manifest list' do
```
Comment (jkeck) [inline spec/views/collections/edit.html.erb_spec.rb]: Is this description correct?

File: spec/views/collections/edit.html.erb_spec.rb [hunk 4]
```diff
@@ -2,12 +2,17 @@
...
+    render
+    expect(rendered).to have_css 'h4', text: 'Workspaces'
+  end
```
Comment (jkeck) [inline spec/views/collections/edit.html.erb_spec.rb]: Is there anyway we can check to see that workspaces were rendered?

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
+  helper ApplicationHelper
```
Comment (jkeck) [inline app/controllers/application_controller.rb]: I'm surprised that this was necessary.  I thought rails did this for us automagically.

File: spec/views/collections/edit.html.erb_spec.rb (hunk 3 above)
Comment (versae) [inline spec/views/collections/edit.html.erb_spec.rb]: Oops. It's not.

File: spec/views/collections/index.html.erb_spec.rb (hunk 2 above)
Comment (versae) [inline spec/views/collections/index.html.erb_spec.rb]: Will do

File: spec/views/collections/edit.html.erb_spec.rb (hunk 4 above)
Comment (versae) [inline spec/views/collections/edit.html.erb_spec.rb]: Now that I know how to create associated objects, I'll check for a specific workspace name.

File: app/controllers/application_controller.rb (hunk 5 above)
Comment (versae) [inline app/controllers/application_controller.rb]: It's not. I'll remove the line.

**Pull request merged**